Makefile

```diff
@@ -7,14 +7,14 @@ install-cq:
 	chmod a+x cloudquery
```

Comment:
Changes to this files were extracted to a separate PR https://github.com/cloudquery/cq-provider-aws/pull/763